pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
+
+- Search the working directory recursively for files containing a regex pattern:
```

Comment:
We commonly use `current`
```suggestion
- Search the current directory recursively for files containing a regex pattern:
```

Reply:
Officially the wording is "working" directory which I prefer, but they mean the same thing. Like the Unix/Linux `pwd` command "returns working directory name" (literally from the man page). But cwd is also used. It's not critical to me either way.

Reply:
If the official technical language is `working`, then it does make sense to keep it as `working` in this instance, though in the interests of consistency with other pages  I think `current` is best in this specific case.

Reply:
Let's use `current` for consistency.